docs/Storage.md

```diff
@@ -81,6 +83,54 @@ storage.destroy()
 
 ```
 
+## Import
+
+Storages can be imported either by passing a URL or by uploading the file. Currently .iso, .raw and .img formats
+are supported. Other formats like qcow2 or vmdk should be converted before uploading.
+
+Uploaded storage is expected to be uncompressed. It is possible to upload zip (`application/gzip`)
+or gzip (`application/x-xz`) compressed files, but you need to specify a separate `content_type`
```

Comment:
Are those mimetypes truly correct?

Reply:
Gracefully off by one. :P Will confirm what is actually supported and edit this before merging.